datasets/hst.yaml

```diff
@@ -0,0 +1,23 @@
+Name: HST public data
+Description: |
+  The Hubble Space Telescope (HST) is one of the most productive scientific instruments ever created. This dataset contains calibrated and raw data for all of the currently active instruments on HST: ACS, COS, STIS and WFC3. 
+Documentation: http://astroquery.readthedocs.io/en/latest/mast/mast.html
+Contact: archive@stsci.edu
+UpdateFrequency: Hourly
+Tags:
+  - astronomy
+License: STScI herby grants the non-exclusive, royalty free, non-transferable, worldwide right and license to use, reproduce and publicly display in all media public data from the Hubble Space Telescope.
+Resources:
+  - Description: Project data files
```

Comment:
Could you make a note that this is a Requester Pays bucket so people know to add the flag as appropriate? Other than that, looks good to go!